### ISSUE TITLE
Color picker configuration reset

### ISSUE THREAD

**Issue by barraganlazlo** [BearishSun/BansheeEngine#278]
![image](https://user-images.githubusercontent.com/24246051/56807712-e5d1e300-682f-11e9-9402-fa3a1ebb7164.png)

Each time i set the color picker mode to SV_H and RGB when i close it and reopen it, it reset to default (BG_R and HSV)
it would be nice if the color picker can save the configuration we last used

**Comment by BearishSun** [comment]
I agree, it would be a nice thing to have. I'll add it one of the releases.
